stock_available_to_promise_release/models/stock_picking.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Camptocamp (https://www.camptocamp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    # Add store on the field, as it is quite used in the searches,
+    # and this is an easy-win to reduce the number of SQL queries.
+    picking_type_code = fields.Selection(store=True)
```

Comment:
@guewen is this still the case? Seems not used that much in queries.

Reply:
Good catch, I removed the search in https://github.com/OCA/stock-logistics-warehouse/pull/709/commits/f25aad87b5642651daecf7823e6654a5f7cf7dcb#diff-4f82f52188d1f3d88f496e52216b1316

It can be removed

Reply:
done